scripts/python/amend_user_repository_access.py

```diff
@@ -620,8 +586,9 @@ def remove_users_with_duplicate_access(github_service: GithubService, repo_issue
                     repository_name != previous_repository
                 ):
                     # raise an issue to say the user has been removed and has access via the team
-                    create_an_issue(
-                        github_service, repo_issues_enabled, username, repository_name)
+                    if repo_issues_enabled.get(repository_name, repo_issues_enabled):
```

Comment:
pulled this `if` statement out of the function. Not entirely sure of its purpose.

But hopefully that becomes more apparent as I refactor more bits 🔧

Reply:
Some repo's do not have Issues enabled to cannot be remember if this causes an error / exception / or was just saving wasted API calls